lib/command_runner.dart

```diff
@@ -147,6 +154,21 @@ class CommandRunner<T> {
     Command? command;
     var commandString = executableName;
 
+    // Returns commands similar to `name`, in sorted order.
+    Iterable<Command<T>> similarCommands(String name) {
+      final distances = Expando<int>();
```

Comment:
Why is this an expando and not just:

```dart
var distances = <String, int>{};
```

?

(Also, we use `var` for all locals in args.)